devices/ePaper/README.md

```diff
@@ -0,0 +1,111 @@
+﻿# nanoFramework.Iot.Device.ePaper
+This is a C# library to bring support for various ePaper displays to nanoFramework. It contains the drivers with a graphics library that provides APIs to draw text, lines, and shapes.
+
+## Currently Supported ePaper Displays
+There are various EPD Display Controllers out there. The following are the Controllers we support. More Controllers will be supported in future releases.
+
+- SSD1681: 200 Source, 200 Gate Red/Black/White Active Matrix EPD Display Driver with Controller.
+
+The datasheets for the supported controllers can be found in `Datasheets`.
+
+Upcoming support:
+- SSD1680: 176 Source x 296 Gate Red/Black/White Active Matrix EPD Display Driver with Controller.
+- UC8151D: All-in-one driver IC w/ Timing Controller for White/Black/Red Dot-Matrix Micro-Cup ESL.
+- And more...
+
+## Circuit
+
+![](ESP32-ePaper-Sketch_bb.png)
```

Comment:
For inclusivity, it's always better to add a description for pictures. Some persons who do not see well uses reading features to descrbe the picture.

```suggestion
![ePaper sketch](ESP32-ePaper-Sketch_bb.png)
```